.github/workflows/style.yml

```diff
@@ -53,7 +53,7 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install -r src/app/requirements.txt --upgrade
-          pip install "mypy==1.4.1" types-requests types-python-dateutil
+          pip install "mypy==1.4.1" types-requests types-python-dateutil types-passlib types-python-jose sqlalchemy-stubs
```

Comment:
Does this mean that these libraries were missing to pass the tests  ? So it solves the current issues within the CI ?

Reply:
Yes, plus the other changes in the code